### PR TITLE
Cache tests memory issues

### PR DESCRIPTION
**Pull request by trasher**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

Sometimes, when running tests on local instance, I ran into memory limit on cache.

### CODE REVIEW

File: inc/config.class.php [hunk 1]
```diff
@@ -2841,12 +2841,8 @@ public static function getCache($optname, $context = 'core', $psr16 = true) {
...
       }
 
-      //use memory adapter when called from tests
-      if (defined('TU_USER') && !defined('CACHED_TESTS')) {
-         $opt['adapter'] = 'memory';
-      }
-      //force FS adapter for translations for tests
-      if (defined('TU_USER') && $optname == 'cache_trans') {
+      //use FS adapter when called from tests
+      if (defined('TU_USER') && (!defined('CACHED_TESTS') ||$optname = 'cache_trans' )) {
```
Comment (cedric-anne) [inline inc/config.class.php]:
```suggestion
      if (defined('TU_USER') && (!defined('CACHED_TESTS') || $optname = 'cache_trans' )) {
```

File: tests/bootstrap.php [hunk 2]
```diff
@@ -33,19 +33,79 @@
...
+      } else {
+         unlink($path);
+      }
+   }
+}
+
+if (!file_exists(GLPI_VAR_DIR)) {
+   mkdir(GLPI_VAR_DIR);
+}
+//cleaup files directory
```
Comment (cedric-anne) [inline tests/bootstrap.php]:
```suggestion
//cleanup files directory
```

File: tests/bootstrap.php [hunk 3]
```diff
@@ -33,19 +33,79 @@
...
+emptyDirectory(GLPI_VAR_DIR);
+
+//crete empty directories
```
Comment (cedric-anne) [inline tests/bootstrap.php]:
```suggestion
//create empty directories
```

File: inc/config.class.php (hunk 1 above)
Comment (trasher) [reply inc/config.class.php]: A "=" was missing also here

**Pull request closed without merging**